internal/bgp/frr/config.go

```diff
@@ -102,6 +102,9 @@ router bgp {{$r.MyASN}}
 {{- if ne .BFDProfile ""}} 
   neighbor {{.Addr}} bfd profile {{.BFDProfile}}
 {{- end }}
+{{- if eq .IPFamily.String "ipv6" }}
```

Comment:
Should we specifically check for ebgp only too?

Reply:
I was debating this with myself too , ebgp is the only type that does directly connection check since by default ebgp nbr needs to one one hop away unless user specify otherwise while for iBGP no such restriction so its kind of no-op 
on the other side its easy to add compare myasn with asn too so not sure WDYT ?

Reply:
I limited it to eBGP only and add more test cases